### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by mofongox** [aeharding/voyager#1458]
### Crash description

So, apparently for no reason, my Voyager app started crashing this morning whenever I touched the inbox icon. 

I tried with and without VPN, custom DNS, automatic DNS, Rethink enabled and disabled, and it just keeps crashing. 

I cleared data, un-installed and reinstalled, and it is still happening.

### Device and app metadata

  - window.location.href: `https://localhost/inbox/all`
  - react-router location.pathname: `/inbox/all`
  - Logged in? `true`
  - Native app? `true`
  - Installed to home screen? `true`
  - Voyager version: `2.8.0`
  - BUILD_FOSS_ONLY: `true`
  - User agent: `Mozilla/5.0 (Linux; Android 13; DE2117 Build/TQ3A.230901.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/122.0.6261.119 Mobile Safari/537.36`

### Crash data

Error: `7:1-22:4: Unknown language: `May` is not registered`

#### Stack trace

```
Error: Unknown language: `May` is not registered
    at t (https://localhost/index.js:247:183199)
    at Object.qq [as highlight] (https://localhost/index.js:247:183387)
    at https://localhost/index.js:247:192650
    at i (https://localhost/index.js:247:191982)
    at node (element<code>) (https://localhost/index.js:247:191541)
    at node (element<pre>) (https://localhost/index.js:247:191695)
    at node (root) (https://localhost/index.js:247:191695)
    at a$e (https://localhost/index.js:247:191214)
    at c$e (https://localhost/index.js:247:191924)
    at https://localhost/index.js:247:192314
```

**Comment by aeharding** [comment]
Thanks for the crash report! This should be solved in 2.8.1, released shortly.

**Comment by mofongox** [comment]
> Thanks for the crash report! This should be solved in 2.8.1, released shortly.

Holy Jesus. Because of this alone I don't dare use a different Lemmy client ever again. Wao!
